tests_v2/unit/app/routes/datasets/test_queries.py

```diff
@@ -36,9 +36,10 @@ async def test_query_dataset_with_api_key(
         f"/dataset/{dataset_name}/{version_name}/query",
         params=params,
         headers=headers,
+        follow_redirects=True,
     )
 
-    print(response.json())
+    # print(response.json())
```

Comment:
I see that these `print` statements are commented out, but in other files, you deleted them entirely. Is there a reason that these are important to keep?

Reply:
Hmm, no. I will remove these entirely as well.